src/cljs/rems/navbar.cljs

```diff
@@ -62,16 +62,15 @@
   ;;TODO: get navigation options from subscription
   (let [roles (:roles identity)]
     [e (into [:div.navbar-nav.mr-auto
-              (when (roles/is-logged-in? roles)
-                [nav-link "/catalogue" (text :t.navigation/catalogue)])
+              (when-not (:user identity)
+               [:<> [nav-link "/" (text :t.navigation/home) :exact] 
+                    [nav-link "/catalogue" (text :t.navigation/catalogue)]])
```

Comment:
This line should have the check for `:catalogue-is-public`, because we don't want to show the page unless you can actually view the contents.

Reply:
Also indentation is wrong here.